app/jobs/base_job.rb

```diff
@@ -75,10 +75,26 @@ def notify_on_exception(exception)
     setup_notifier('DelayedJobFailure')
     return unless @send_notifications
 
+    log_group_name = ENV.fetch('TARGET_GROUP_NAME', nil)
+    log_stream_name = ENV.fetch('LOG_STREAM_NAME', nil)
+
+    if log_group_name.nil? || log_stream_name.nil?
+      Rails.logger.info 'TARGET_GROUP_NAME is not set.' if log_group_name.nil?
+      Rails.logger.info 'LOG_STREAM_NAME is not set.' if log_stream_name.nil?
+    else
+      log_stream_escaped = log_stream_name.gsub('/', '$252F')
+      log_stream_link = "https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:log-groups/log-group/#{log_group_name}/log-events/#{log_stream_escaped}"
+    end
+
     msg = [
       "*#{self.class.name}* `FAILED` with the following error:",
       "```\n #{exception.inspect} \n```",
-    ].join("\n")
+    ]
+
+    msg << "Log stream: #{log_stream_name}" unless log_stream_name.nil?
```

Comment:
Instead of adding the text to the end of the message let's do it more like this:
```request.env['exception_notifier.exception_data'] = {
      current_user: current_user&.email || 'none',
      current_user_browser: browser.to_s,
    }```